standalone_tests/image_bounding_box.py

```diff
@@ -69,6 +69,19 @@ def gen_box_img_type_2(vrange=1, box_count=100):
     box_img = wandb.Image(image, boxes=[gen_box_type_2(vrange) for i in range(1,box_count)] )
     return box_img
 
+class_id_to_label = {
+    0: "car",
+    1: "truck",
+    2: "boat",
+    3: "cthululu",
```

Comment:
cthulhu, nice one! got me to look up and learn a new spelling!

Reply:
The h is much more guttural